Sources/Controllers/Settings/ImportExport/OAImportSettingsViewController.mm

```diff
@@ -22,35 +22,32 @@ @implementation OAImportSettingsViewController
     NSArray<OASettingsItem *> *_settingsItems;
     BOOL _isNewItems;
     NSString *_file;
-    NSString *_descriptionText;
-    NSString *_descriptionBoldText;
     QList<OsmAnd::ArchiveReader::Item> _archiveItems;
-    NSString *_headerLabel;
+
+    BOOL _importStarted;
```

Comment:
It is better to call backend to see if import started or not. `_importStarted` field looks bad.